app/frontend/src/utils/permissionUtils.js

```diff
@@ -45,3 +47,41 @@ export function checkSubmissionView(userForm) {
   ];
   return userForm && userForm.permissions && userForm.permissions.some(p => perms.includes(p));
 }
+
+/**
+ * @function determineFormNeedsAuth
+ * When loading a form to fill out, determine if the user needs to log in to submit it
+ * @param {Object} store The vuex store reference
+ * @param {String} formId The form guid
+ * @param {String} formId The submissionId guid
```

Comment:
@param {String} submissionId The submission guid

Reply:
👍